std/haxe/io/Bytes.hx

```diff
@@ -300,7 +300,7 @@ class Bytes {
 	}
 
 	/**
-		Returns the 16 bit unsignged integer at given position (in low endian encoding).
+		Store the 16 bit unsigned integer at given position (in low endian encoding).
```

Comment:
We should make up our minds about "store" vs. "stores" throughout the documentation. I think I used third person everywhere.